Src/adc_interface.c

```diff
@@ -99,7 +99,7 @@ uint8_t Set_Cell_One_Voltage(uint32_t adc_reading) {
 		return 0;
 	}
 
-	adc_values.cell_1_voltage = (adc_reading * CELL_ONE_ADC_SCALAR) + CELL_ONE_ADC_OFFSET;
+	adc_values.cell_1_voltage = (adc_reading * CELL_ONE_ADC_SCALAR) - CELL_ONE_ADC_OFFSET;
```

Comment:
This starts getting confusing have some as addition and others as subtraction. I'd rather they all be addition in the math then have signed ints as #defines.